modules/olmv1-installing-an-operator.adoc

```diff
@@ -50,7 +50,7 @@ $ oc apply -f pipeline-operator.yaml
 .Example output
 [source,text]
 ----
-clusterextension.olm.operatorframework.io/pipelines-operator created
+clusterextension.olm.operatorframework.io/pipes created
```

Comment:
@michaelryanpeter it is clusterextension.olm.operatorframework.io/pipelines-operator because of https://github.com/openshift/openshift-docs/pull/88848/files#diff-57d7e8d28879735eab92932f9cc5f093124acaf00427fc24c2ce84e78e186d99R26

Reply:
Addressed: https://github.com/openshift/openshift-docs/pull/88848/files#diff-57d7e8d28879735eab92932f9cc5f093124acaf00427fc24c2ce84e78e186d99R57